readme.md

```diff
@@ -39,7 +39,7 @@ A complete list of inputs, including changes made to the framework since the ori
 [here](http://gfw2-data.s3.amazonaws.com/climate/carbon_model/Table_S3_data_sources__updated_20230406.pdf).
```

Comment:
Delete "A complete list of inputs, including changes made to the framework since the original publication, can be found 
[here](http://gfw2-data.s3.amazonaws.com/climate/carbon_model/Table_S3_data_sources__updated_20230406.pdf).". I'm not updating this anymore.